\chapter{The Polar Front as a major biogeographic boundary in the Southern Ocean} 
\label{ch:polarfront}

Sections of this chapter have been previously published in \bibentry{Wilkins:2012td}.

\section{Summary}

\section{Introduction}


\section{Methods}
\subsection{Sampling and metagenomic sequencing}

Sampling\footnote{Sampling was performed by Jeffrey M. Hoffman and Jeffrey B. McQuaid} was conducted on board the RSV \emph{Aurora Australis} during cruise V3 \ac{CEAMARC/CASO} from 13 December 2007 -- 26 January 2008. 
This cruise occupied the SR3 latitudinal transect from Hobart, Australia (44\textdegree{} S) to the Mertz Glacier, Antarctica (67\textdegree{} S) within a longitudinal range of 140--150\textdegree{} E.
Nineteen samples (16 surface, 3 deep) were obtained along almost the entire latitudinal range \figref{fig:samplemap}.

\input{../polarfront/samplemap.tex}

A range of data were recorded by integrated instruments on the RSV \emph{Aurora Australis} including location, water column depth, water temperature, salinity, fluorescence and meterological data \tabref{tab:samplelist}.
These data were used to locate the \ac{PFZ} based on a surface temperature gradient of $\sim$ 1.35 \textdegree{}C across a distance of 45--65 km, placing the \ac{PF} at approximately $-59.70$\textdegree{} of latitude, consistant with previous descriptions \cite{Moore:1999to,Sokolov:2002tc}.
Samples were accordingly grouped into ``North'' and ``South'' zones, while the three deep samples composed a ``Deep'' zone \tabref{tab:samplelist}.
The \ac{NZ} represents waters from the Subtropical, Subantarctic and \ac{PFZ} regions, while the \ac{SZ} represents the \ac{AZ}.

\input{../polarfront/samplelist.tex}

At each station, $\sim$ 250--560 L of seawater was pumped from $\sim$ 1.5--2.5 m below the sea surface into drums stored at ambient temperature on deck. 
In the case of deep samples, $\sim$ 225--230 L of seawater was collected from Niskin bottles attached to a \ac{CTD} (SeaBird, Bellevue, USA).
Seawater samples were prefiltered through a 20 \micron plankton net, then filtrate was captured on sequential 3.0 \micron 0.8 \micron and 0.1 \micron 293 mm polyethersulfone membrane filters (Port Washington, USA), and immediately stored at $-20$ $^\circ$C \cite{Rusch:2007ez,Ng:2010cd}.

DNA extraction\footnote{DNA extraction was performed by Cynthia Andrews-Pfannkoch and others at the J. Craig Venter Institute} was performed at the J. Craig Venter Institute (Rockville, USA) as described in \citet{Rusch:2007ez}.
Pyrosequencing was performed on a GS20 FLX Titanium instrument (Roche, Branford, USA) also at the J. Craig Venter Institute as described in \citet{Lauro:2010jna}.
Duplicate reads and reads with many pyrosequencing errors were removed as described in \citet{Lauro:2010jna}.

\subsection{Phylogenetic analysis of metagenomic data}

\subsubsection{\softwarename{blast} comparison to RefSeq database}

A subset of the RefSeq microbial (bacterial and archaeal) genome database (release 41, retrieved May 31 2012 from \url{ftp://ftp.ncbi.nih.gov/refseq/release/}) was prepared by excluding sequences with the words ``shotgun'', ``contig'', ``partial'', ``end'' or ``part'' in their headers \cite{Angly:2009ip}.
Because this database was not expected to contain representative genomes for every species present, \acp{OTU} in this study are defined by the best species match to this database, and may for example represent congeners.

The metagenomic reads from each sample were compared against this database using \softwarename{tblastx}, with default parameters except for: E-value threshold $1.0\times{}10^{-3}$, cost to open gap 11, cost to extend gap 1, masking of query sequence by \softwarename{SEG} masking with lookup table only.
The outputs of all \softwarename{tblastx} searches against RefSeq were processed by \softwarename{minspec} (see section following), and hits not belonging to the minimal sets were removed.

\subsubsection{Identification of minimal species sets with \softwarename{minspec}}

A computational method to minimise false \ac{OTU} identifications and increase the accuracy of \ac{OTU} abundance estimates (\softwarename{minspec}) was developed and implemented in \softwarename{perl}\footnote{\softwarename{minspec} and the associated metagenomic simulation and validation scripts are open source and available at \url{https://github.com/wilkox/minspec}; a copy has also been provided in the supplementary information.}.
Following the approach of \citet{Ye:2009bl} to the parsimonious reconstruction of biochemical pathways (\softwarename{MinPath}), \softwarename{minspec} computes the smallest set of OTUs sufficient to explain a set of observed high-quality hits against RefSeq (or any other sequence database).
The minimal set computation is framed as a linear programming problem and solved with the \ac{GLPK} tool \ac{GLPSOL} (Free Software Foundation, Boston).
This approach eliminates many of the spurious \ac{OTU} identifications which result from reads with strong identity to more than one \ac{OTU}. 
The ``minimal species set'' is liable to exclude some low-abundance \acp{OTU}, but gives more faithful abundance estimates and eliminates many false positives.

To validate this approach and estimate error rates, simulated microbial assemblages were generated and simulated metagenomic sampling and \softwarename{blast} search was performed on each assemblage.
To simulate sequence identity between taxa, each simulated taxon went through up to fifty rounds in which another taxon was selected at random and deemed to have sequence identity with the first.
After each round, the this process was terminated with a 10\% probability to simulate an exponential curve of interrelatedness between taxa.
A random subset of the simulated taxa were then selected to form the simulated assemblage.
This allowed for the possibility of taxa in the assemblage having ``sequence identity'' to taxa outside it, thus representing the problem \softwarename{minspec} was designed to mitigate.
A simulated \softwarename{blast} search was then performed, in which a taxon was selected at random to generate a \softwarename{blast} hit.
To represent a lack of species evenness, taxa were more or less likely to produce a hit according to a logarithmic relationship with their rank within the assemblage, producing a naturalistic rank-abundance curve.
Each time a taxon was selected to produce a hit, other taxa with simulated sequence identidy were also selected to produce hits for that ``read'', again simulating the problem of a single read producing multiple hits to closely related taxa.

To fully explore the limits and reliability of \softwarename{minspec}, the simulated metagenomic experiment described above was performed with all possible permutations of the following parameters: number of simulated taxa [100, 1000, 10000, 50000, 100000]; size of simulated assemblage [1, 10, 100, 300, 500, 1000, 10000]; number of simulated metagenomic reads [10, 100, 1000, 10000, 100000, 200000].
Each permutation was repeated five times, except for those where the size of the assemblage would exceed the number of taxa simulated.
The resulting simulated \softwarename{blast} outputs were processed with \softwarename{minspec}, and the false positive (percentage of taxa not in the assemblage which nevertheless survived \softwarename{minspec} filtering) and false negative (percentage of taxa present in the assemblage which were not present after minspec filtering) rates calculated.
Because a high false negative rate can arise from undersampling, a problem in metagenomic studies both real and simulated, an additional ``false negative (\softwarename{minspec})'' metric was calculated, which excluded taxa which were present in the assemblage but through random chance did not generate any reads, the equivilant of ``unsampled rare taxa''.
This rate thus represented only false negatives attributable to \softwarename{minspec} itself.
Finally, as a measure of \softwarename{minspec}'s usefulness, the proportion of ``false'' taxa --- those which generated \softwarename{blast} hits but were not part of the assemblage --- that were succesfully removed by \softwarename{minspec} was calulated.

\subsubsection{\ac{OTU} abundances and variance between zones}

The relative \ac{OTU} abundances for each sample were determined using the \softwarename{perl} script \ac{GAAS} \cite{Angly:2009ip}.
Briefly, \ac{GAAS} estimates the relative abundance of \acp{OTU} from the number and quality of BLAST hits to each species, taking into account differences in genome size. 
\ac{GAAS} was run with the default settings. 
To normalise for reads which did not yield acceptable hits, the relative abundances for each sample were scaled by that sample's effective \softwarename{blast} hit rate. 
An \ac{OTU} profile was generated for each sample by encoding the scaled relative abundance of each \ac{OTU} from each size fraction as a separate variable.

To test the hypothesis that the oceanic zones harbour significantly different communities, \ac{ANOSIM} with 999 permutations was performed on a standardised, log-transformed Bray-Curtis resemblance matrix of \ac{OTU} profiles.
\ac{SIMPER} analysis was performed to identify the contribution of individual \acp{OTU} to differences between the zones. 
All statistical procedures were performed in \softwarename{PRIMER 6} as described by \citet{Clarke:2001ut}.

\subsection{Functional analysis of metagenomic data}

\subsubsection{\softwarename{blast} comparison to \ac{KEGG} database}

In order to identify functional differences between the zones, the set of metagenomic reads from each sample was compared against the \ac{KEGG} GENES database (retrieved July 2 2010 from \url{ftp://ftp.genome.jp/pub/kegg/genes/fasta/genes.pep}) with \softwarename{blastx}, with default parameters except for: maximum number of database sequence alignments 10; E-value threshold $1.0\times10^{-3}$; gap opening penalty 11; gap extension penalty 1; masking of query sequence by \softwarename{SEG} masking for lookup table only.

\subsubsection{Analysis of functional potential}

Genes identified by \softwarename{blastx} were aggregated to \ac{KEGG} ortholog groups according to the \ac{KEGG} Orthology schema (\url{ftp://ftp.genome.jp/pub/kegg/genes/ko}, retrieved Mar 29 2011), and ortholog group abundances calculated for each sample. 
Following \citet{Coleman:2010jj}, a read was considered a hit to a given ortholog group if the top three hits for that read (or all hits if fewer than three total hits) were to genes from the same ortholog group, and had bit scores \textgreater{} 40. 
If the bit score difference between any two top hits was greater than 30, only the hits above this difference were considered.

Ortholog group counts were then used to calculate the abundance of KEGG modules.
Because many ortholog groups are members of more than one module, the abundance $a_m$ of each module $m$ was calculated as 
\[
a_{m}=\sum_{K=1}^{n}\frac{C_{K}}{M_K}
\]
where $n$ is the number of ortholog groups $K$ belonging to module $m$, $C_{K}$ is the number of hits to ortholog group $K$, and $M_{K}$ is the total number of modules to which $K$ belongs.
To account for differences in sequencing depth between samples, module abundances were scaled to 500,000 reads per sample. 
To test the hypothesis that the \ac{NZ} and \ac{SZ} harbour significantly different functional potential, one-way \ac{ANOSIM} with 999 permutations was performed as above on a standardised, log-transformed Bray-Curtis distance resemblance matrix of the module and ortholog group profiles. 
A functional profile was generated for each sample by summing the scaled abundances of each module from all size fractions, and \ac{SIMPER} performed as above to identify modules which contributed highly to the variation in functional potential between the two zones. 
Modules with a high contribution to variance or otherwise of interest were then linked to taxonomy (``taxonomic decomposition'') by noting the genus of the organism associated with each gene in the \ac{KEGG} GENES database and thus calculating the relative contribution of each genus to each module's abundance. 
This allowed functional contributions to be putatively assigned to genera which were not identified in our taxonomic analysis, as the database included gene sequences for organisms for which a full genome was not available.

\section{Results}

\subsection{Metagenomic sequencing}
6.6 Gbp of 454 sequence data representing picoplankton in the size range 0.1 -- 3.0 \micron was obtained from 16 samples. 
After removal of low-quality reads, 454 sequencing yielded 157,507 -- 597,689 reads per sample (mean 354,399) of lengths ranging from 100 to 606 bp (mean 378).

\subsection{Phylogenetic analysis of metagenomic data}

The proportion of reads in each sample which yielded matches to RefSeq ranged from 25\% to 85\% (mean 62\%).
The most abundant \acp{OTU} in each sample are given in \tabreft{tab:topotus} and a full list of \ac{OTU} abundances in the supplementary material \suppfile{PF-all-OTUs.csv}.
All samples and size fractions exhibited very low \ac{OTU} evenness \figref{fig:rankabundance}.

\input{../polarfront/topotus.tex}
\input{../polarfront/rankabundance.tex}

\ac{ANOSIM} analysis showed that the zones harbor significantly different microbial communities (R = 0.451, p < 0.004). 
\ac{SIMPER} was performed in order to identify the contribution of individual \acp{OTU} to the difference between the \ac{NZ} and \ac{SZ}. 
The results for the highest contributors are provided in \tabreft{tab:otussimper}, and are graphically summarised for all \acp{OTU} in \figreft{fig:taxotreemap}.

\input{../polarfront/otussimper.tex}
\input{../polarfront/taxotreemap.tex}

The \ac{SIMPER} analysis found that no single \ac{OTU} contributed more than 2.9\% of variance and 74\% of variance was contributed by \acp{OTU} with a contribution less than 1\%. 
There was also a large difference in the contribution to variance of the three size fractions, with approximately 52\% of all variance contributed by \acp{OTU} from the 3.0 \micron{} fraction, 37\% by the 0.8 \micron{} fraction, and 9\% by the 0.1 \micron{} fraction.
Notably, \acp{OTU} within several taxonomic groups that had high contribution to variance covaried in their relative representation in the \ac{NZ} and \ac{SZ}.
For example, Bacteroidetes and GSO-EOSA-1 representatives were on average more abundant in the \ac{SZ}; while \emph{Prochlorococcus} and \emph{Synechococcus} spp., SAR11 and SAR116 were on average more abundant in the \ac{NZ} \figref{fig:taxotreemap}.
Some groups, such as the Alteromonadales, had variable relative representation depending on size fraction.

\subsubsection{Validation of \softwarename{minspec}}
TODO working on this section
Here is a ref to the figure \figref{fig:minspecvalidation}.
And for fun, the false positives only \figref{fig:minspecvalidationfalsepositive}.

\input{../polarfront/minspecvalidation.tex}


\subsection{Functional analysis of metagenomic data}

TODO working on this section

\ac{ANOSIM} analysis of the samples' \ac{KEGG} ortholog group and module profiles revealed that the zones had significantly different functional potential (ortholog group: R = 0.642, p < 0.001; module: R = 0.871, p < 0.001). 
\ac{SIMPER} was performed on the profiles in order to identify the specific functional differences between the zones. 
The highest-contributing modules are given in \tabreft{tab:modulessimper}, and a complete list in the supplementary material \suppfile{PF-modules-SIMPER.csv}.
The highest-contributing ortholog groups are given in \tabreft{tab:orthologsimper}, and a complete list in the supplementary material \suppfile{PF-ortholog-groups-SIMPER.csv}.
No single ortholog group or module contributed more than 2.2\% of the variance, indicating a complex and diverse pattern of functional differences. 
There was a strong trend for ortholog groups and modules with higher contributions to variance to be overrepresented in the \ac{NZ} in the 3.0 \micron{} fraction but the \ac{SZ} in the smaller fractions, indicating that the functional diversity of each zone was strongly segregated by size fraction.

\input{../polarfront/modulessimper.tex}
\input{../polarfront/orthologsimper.tex}

\section{Discussion}

\section{Conclusions}

